package.json

```diff
@@ -17,6 +17,7 @@
     "less-loader": "^5.0.0",
     "lodash": "^4.17.19",
     "luxon": "^1.24.1",
+    "nodemailer": "^6.4.11",
```

Comment:
This was installed with `npm install nodemailer --save`. We can add this as a dev dependency using `npm install nodemailer --save-dev`. The main idea here is that the website doesn't need `nodemailer`. It's just a tool used on the local developer's machine. So it doesn't need to be included as a `dependency`, which will just bloat the website. Dev dependencies allow us to install these tools without bloating the website